karma-shared.conf.js

```diff
@@ -10,8 +10,16 @@ module.exports = function(config, specificOptions) {
     browserDisconnectTimeout: 10000,
     browserDisconnectTolerance: 2,
     browserNoActivityTimeout: 30000,
-
-
+    reporters: ['spec'],
+    specReporter: {
+      maxLogLines: 5,             // limit number of lines logged per test
+      suppressErrorSummary: true, // do not print error summary
+      suppressFailed: false,      // do not print information about failed tests
```

Comment:
Exactly. "suppressFailed: true" would hide the failures. "suppressErrorSummary: true" is set because the summary is basically a list of all errors printed after all tests have run - not very useful if you only show the errors anyway.
We should probably increase maxLogLines to see more of failures with good stack traces.